CODEOWNERS

```
@@ -1,2 +1,2 @@
 
-.github/workflows   @mozilla/xr
+.github/workflows   @Hubs-Foundation
```

Apparently GitHub doesn't like this, so just use the Operations team for now.  We should probably revisit stuff like this later on a larger scale and decide how we want to handle it.
```suggestion
.github/workflows   @Hubs-Foundation/operations 
```